project/vision_backend/queues.py

```diff
@@ -0,0 +1,143 @@
+import abc
+import json
+import logging
+import posixpath
+import time
+
+import boto.sqs
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+from django.utils.module_loading import import_string
+from six import StringIO
```

Comment:
Just so you know, you can use `from io import StringIO` instead. That actually goes for Python 2.7 as well as 3.x. If this code works though, then we can leave it for now. I'll clean up all the `six` imports together at some point, anyway.

Reply:
I see. Yeah, this import was a left-over from your "swappable backend" branch. I'll just leave it for now and we can purge along with the other when we deprecate 2.7.

Reply:
Ah, I see. Right, I didn't understand all the Python 2 vs. 3 imports as well when I made that branch.